### DISCUSSION TITLE
de novo transcriptome assembly approach

### DISCUSSION THREAD

**Discussion by ConcettaDe4** [ablab/spades#739] [Q&A]
Hi all!

I want to perform the de novo transcriptome assembly of two cultivars of the same plant species and after I want to perform a differential expression analysis between the two cultivars. In particular, I have RNA-seq library produced from two cultivars of the same species.

I would like to ask how to perform the de novo transcriptome assembly.

I was wondering if I should assemble together the two cultivars merging in a unique dataset all RNA-seq libraries or it is better to perform two separate assembly, one for each cultivar and subsequently merge together the two assembly.

How the two approaches will affect the transcriptome assembly and the identification of isoforms?

Thank you!

Concetta

**Comment by andrewprzh** [comment]
Dear Concetta

I'd suggest to use both samples together and use some standard tool for transcript quantification, e.g. kallisto.
This way you'll get more coverage and thus, likely a more complete assembly.

Of course, alternative isoforms may complicate the assembly. However, rnaSPAdes algorithms are designed to resolve such cases, so I believe benefit from higher overall coverage will be greater.

All the best
Andrey
